wp-frontend-profile.php

```diff
@@ -3,7 +3,7 @@
  * Plugin Name: WP Frontend Profile
  * Plugin URI: https://wordpress.org/plugins/wp-front-end-profile/
  * Description: This plugin allows users to easily edit their profile information on the frontend rather than having to go into the dashboard to make changes to password, email address and other user meta data.
- * Version:     1.3.2
+ * Version:     1.3.3
```

Comment:
Require to change version for **WPFEP_VERSION**

Reply:
@musamamasood fixed.